server/base/src/main/java/org/apache/accumulo/server/util/MetadataTableUtil.java

```diff
@@ -894,12 +895,19 @@ public static void removeBulkLoadEntries(Connector conn, String tableId, long ti
         BatchWriter bw = conn.createBatchWriter(MetadataTable.NAME, new BatchWriterConfig())) {
       mscanner.setRange(new KeyExtent(tableId, null, null).toMetadataRange());
       mscanner.fetchColumnFamily(TabletsSection.BulkFileColumnFamily.NAME);
+      boolean shouldTrace = log.isTraceEnabled();
+      byte[] tidAsBytes = Long.toString(tid).getBytes(UTF_8);
       for (Entry<Key,Value> entry : mscanner) {
-        log.debug("Looking at entry " + entry + " with tid " + tid);
-        if (Long.parseLong(entry.getValue().toString()) == tid) {
-          log.debug("deleting entry " + entry);
-          Mutation m = new Mutation(entry.getKey().getRow());
-          m.putDelete(entry.getKey().getColumnFamily(), entry.getKey().getColumnQualifier());
+        if (shouldTrace) {
+          log.trace("Looking at entry " + entry + " with tid " + tid);
```

Comment:
This is not related to this PR, but its something I learned about slf4j vs log4j.  This conversation reminded me of it and I thought it was worth sharing.

Varargs methods can be expensive in a tight loop, because it allocates an array and copies to it.  Slf4j avoids this for one and two arguments by providing specialized methods.  However 3 or more args will use varargs.  

I learned from @EdColeman that log4j V2  provides specialized methods for up to 10 arguments to avoid varargs. For example this is the [debug method that takes 10 args](https://logging.apache.org/log4j/2.x/log4j-api/apidocs/org/apache/logging/log4j/Logger.html#debug-org.apache.logging.log4j.Marker-java.lang.String-java.lang.Object-java.lang.Object-java.lang.Object-java.lang.Object-java.lang.Object-java.lang.Object-java.lang.Object-java.lang.Object-java.lang.Object-java.lang.Object-)

Wish slf4j had this.